scripts/help

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+echo "Description: Makefile to build, test and package Harvester
+Documentation: https://github.com/harvester/harvester/wiki#build--development
+Source: https://github.com/harvester/harvester/tree/master/scripts
+Maintainer: @harvester/maintainer
+
+Targets:"
+
+shift
+for target in "${@}"; do 
+        desc=$(grep -m1 '^# DESC:' "scripts/${target}" | sed 's/^# DESC:[[:space:]]*//')
+        if [ -z "${desc}" ];
+  then 
+                desc="(no description)"
+        fi;
+        printf "  \033[36m%-20s\033[0m %s\n" "${target}" "${desc}"
```

Comment:
nit: can we fix the indentation for consistency? most other scripts used 2 spaces - thanks.

Reply:
Done :)